docs/CTPUSHTEMPLATES.md

```
@@ -147,11 +147,17 @@ pt_product_display_linear | Optional | `true`
 
 ## Five Icons Template
 
-Five icons template is a push notification with no text, just 5 icons which can help your users go directly to the functionality of their choice with a button's click.
+## Five Icons Template
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)